test/Gate_test.jl

```diff
@@ -74,6 +86,15 @@
             P = parameters(Op)
             @test Op(1:N...; rand(P)...) isa Gate{Op,N,P}
         end
+
+        # Special case for SU{N}
+        for N in [2, 4, 8]
+            _lanes = range(1, length = log2(N) |> Int)
+            random_values = rand(ComplexF32, N, N)
+            q, _ = qr(random_values)
+            _values = Matrix{ComplexF32}(q)
+            @test SU{N}(_lanes...; values = _values) isa Gate{SU{N},log2(N) |> Int, NamedTuple{(:values, :N),Tuple{Matrix,Int}}}
+        end
```

Comment:
This is failing on Julia nightly.

Reply:
Now this should be fixed. The problem was that `ComplexF32` was not precise enough.